moving-forward-series/simple-payment-verification.md

```diff
@@ -0,0 +1,31 @@
+# What is Simple Payment Verification (SPV)  - Moving forward (better title needed)
```

Comment:
What should I change the title to?

Reply:
`Can a Crypto Wallet be Light AND Secure? SPV Done Right - Moving Forward`

Just a quick one, better ideas welcome

Reply:
`Can a Crypto Wallet be Lightweight, Private AND Secure? The SPV Model Leads the Way"

Reply:
When I've finished with a comment, do I press the "Resolve conversation" button or is there something else I should do first?

Reply:
"Resolve conversation" is fine! Anyone can unresolve it if they really wanted to 😂

Reply:
When you "resolve" it collapses the discussion to take less space and mark it as "done" or "processed". All comments are still accessible if you expand, and it can be unresolved if needed. Just an attention management tool.